tests/data/powerwalker-vfi_20kcpgpf133.json

```diff
@@ -0,0 +1,1159 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.8072.3.2.10",
+                    "sysDescr": "Linux SNMP-System",
+                    "sysContact": "<private>",
+                    "version": "2767,10,2738,11.\n53 31 00 00 00 00 00 00 00 00 00 00 00 00 00 00",
```

Comment:
This doesn't look correct

Reply:
You have much more experience than me, could you help me with this? These devices have additional nms cards and they are based on Linux. Unnecessary on owerview tab shows processors ram and starage.